src/security/pom.xml

```diff
@@ -1,59 +1,61 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
+
 <!-- 
  Copyright (C) 2014 - Open Source Geospatial Foundation. All rights reserved.
  This code is licensed under the GPL 2.0 license, available at the root
  application directory.
  -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.geoserver</groupId>    
-    <artifactId>geoserver</artifactId>
-    <version>2.19-SNAPSHOT</version>
-  </parent>
+    <parent>
+        <groupId>org.geoserver</groupId>
+        <artifactId>geoserver</artifactId>
+        <version>2.19-SNAPSHOT</version>
+    </parent>
 
-  <groupId>org.geoserver</groupId>
-  <artifactId>gs-security</artifactId>
-  <packaging>pom</packaging>
-  <name>GeoServer Security Modules</name>
+    <groupId>org.geoserver</groupId>
+    <artifactId>s-security</artifactId>
```

Comment:
Is this from the tooling or manual edits? Certainly looks a bit strange, and appears to be (part of) what is breaking the build.

Reply:
Hum... wondering if the plugin is safe to parallel builds

Reply:
Indeed I'm getting other failures as well. Reported here:
https://github.com/revelc/formatter-maven-plugin/issues